src/pages/search/registration_search/RegistrationSearchBar.tsx

```diff
@@ -53,6 +60,21 @@ export const RegistrationSearchBar = ({ registrationQuery }: Pick<SearchPageProp
 
   const initialSearchParams = createSearchConfigFromSearchParams(searchParams);
 
+  function processSearchParamProperties(
```

Comment:
Nice å samle dette en plass! 🙌 
Kunne vurdert å lage egen `interface` for `values` typen også, og bruke den både her og for `initialSearchParams`, men vi har like sterke typer sånn det står nå også, så det går egentlig for det samme

Reply:
Done 👍